test/List.spec.js

```diff
@@ -1,7 +1,20 @@
-import { shallow } from 'vue-test-utils'
+import { shallow, mount } from 'vue-test-utils'
 import List from '@/components/List.vue'
 
 describe('List.vue', () => {
+  it('should throw an error here', () => {
```

Comment:
We're using Jest's expect assertion library, not chai.

It provides a more convenient [expect.toThrow](http://facebook.github.io/jest/docs/en/expect.html#tothrowerror).

Reply:
Thank-ee, i pushed up another spec that uses toThrow.